lib/groups/help.js

```diff
@@ -20,6 +20,16 @@ class HelpGroup {
         process.stdout.write(`\nwebpack ${webpack.version}\n`);
     }
 
+    outputCommand(command) {
```

Comment:
I think that we can intertwine this with the `outputHelp` method, so we avoid duplicate functionality?

Reply:
Ok!
Just one concern functionalites of both differs alot (one focusing on options and flags, others on descriptions and examples ) and thus duplication shouldn't be a concern imo.
Thanks.

Reply:
I think that it should be one, maybe abstracted inside the `outputHelp` method itself when you validate in `bootstrap.js`. It's more clean as an abstraction rather than adding more duplicate code

Reply:
Okay, did that in latest commit.